server/providers/uploads/local/index.ts

```diff
@@ -29,6 +29,7 @@ export default class LocalProvider implements Provider {
   private uploadMap: Map<string, Upload>
 
   constructor (options: LocalProviderOptions, app: FastifyInstance) {
+    options = options ?? {}
```

Comment:
This should be handled when the backend is loaded. The argument to the constructor is not nullable, so from a TS perspective this is redundant. The provider loader should handle making sure the providers do not receive `undefined` as an argument.